scripts/ci/_utils.sh

```diff
@@ -136,6 +136,14 @@ function initialize_common_environment {
             "--env" "PYTHONDONTWRITEBYTECODE" \
         )
     fi
+
+    if [[ ${CI:=} == "true" ]]; then
```

Comment:
For those curious ones. This is a recommended way of passing codecov variables to docker run command: https://docs.codecov.io/docs/testing-with-docker